examples/port-2-alibaba2-connection/README.md

```diff
@@ -0,0 +1,69 @@
+# Fabric Port to Fabric Alibaba Profile Connection
```

Comment:
Rename folder to `port-2-alibaba-connection-with-alicloud-terraform-integration`

Reply:
Addressed with commit: [e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb](https://github.com/equinix/terraform-equinix-fabric/pull/155/commits/e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb)